app/controllers/parties_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
     @party = Party.find_by(acronym: params[:acronym].upcase)
     # cand.district has downcase and unicode_normalize filters so that the sort is accent insensitive.
     # e.g. Évora was showing up as last
-    @candidates = @party.candidates.where(position: 1).sort_by { |cand| cand.district.downcase.unicode_normalize(:nfd) }
+    @candidates = @party.candidates.where(is_lead_candidate: true).sort_by { |cand| cand.district.downcase.unicode_normalize(:nfd) }
```

Comment:
Why have you change it back?

Reply:
It was actually Vasco who reminded me on this.
I'm having an issue locally where my data isn't getting properly populated.
By adding the filter `is_lead_candidate: true` I don't get any candidates data.

Although, if I filter with `position: 1`, I get data to work with, but it's incorrect data. To give a strong example, Aliança party doesn't list Pedro Santana Lopes.

I changed it back to position: 1, because I'm afraid that it can produce incorrect data on the live website. But I can change it back if we you sure it will work 👍

Reply:
I've checked and you did right, `position: 1` is wrong.